ui_testing/pages/analyses_page.py

```diff
@@ -10,3 +10,8 @@ def get_analysis_page(self):
         self.base_selenium.get(url=self.analyses)
         self.wait_until_page_is_loaded()
 
+    def filter_by_analysis_number(self, filter_text):
+        self.info('Filter by analysis number : {}'.format(filter_text))
+        self.open_filter_menu()
+        self.filter_by(filter_element='analysis_page:analysis_no_filter', filter_text=filter_text, field_type='text')
+        self.filter_apply()
```

Comment:
this function in the orders_page so why you put it in the analyses_page one more time

Reply:
cause filter element in order page differs from analysis page element.